### ISSUE TITLE
Error due to reassigning constant "currentTheme"

### ISSUE THREAD

**Issue by MichaelJHaywood** [dingraham/cypress-runner-themes#1]
An error is occurring due to the reassigning of the "currentTheme" constant. I'm unable to get themes to apply. 

```
X [ERROR] Cannot assign to "currentTheme" because it is a constant

    node_modules/cypress-runner-themes/src/utils.js:26:6:
      26 │       currentTheme = "colorblind";
         ╵       ~~~~~~~~~~~~

  The symbol "currentTheme" was declared a constant here:

    node_modules/cypress-runner-themes/src/utils.js:22:10:
      22 │     const currentTheme = Cypress.env("theme");
         ╵           ~~~~~~~~~~~~
```

**Comment by dingraham** [comment]
@MichaelJHaywood  Yup, easy miss and fix. Thanks for pushing up the change.
I'm at a work conference and will publish when I get back in a few days. Appreciate you!

**Comment by dingraham** [comment]
Fixed in v1.0.2
